components/constellation/Cargo.toml

```diff
@@ -40,6 +41,7 @@ net = {path = "../net"}
 net_traits = {path = "../net_traits"}
 profile_traits = {path = "../profile_traits"}
 script_traits = {path = "../script_traits"}
+script = { path = "../script"}
```

Comment:
This shouldn't be necessary. We can do this from `components/servo/Cargo.toml` instead.

Reply:
I don't completely understand what you re suggesting. Removing this will throw an error and I don't see `../servo` in dependencies. There is `../constellation` in dependencies of  `/components/servo` but how does that help?

Reply:
components/servo/Cargo.toml already has a script dependency in it that we can use to propagate the refcell_backtrace feature. We shouldn't need to modify the constellation Cargo.toml at all.

Reply:
Just to clear it in my mind, `components/servo` will propagate it to `components/constellation` because the latter is a dependency of former, right? :sweat_smile:

Reply:
No; components/servo will propagate it to components/script.

Reply:
I would not expect to see a refcell_backtrace feature in components/constellation/Cargo.toml, so there should be no errors.

Reply:
Okay. I just re-read the cargo feature link you had mentioned in the issue that this PR addresses and things are now much clear to me.